model/file.go

```diff
@@ -22,6 +22,36 @@ type FileUploadResponse struct {
 	ClientIds []string    `json:"client_ids"`
 }
 
+const (
+	FILE_SORT_BY_CREATED      = "CreateAt"
+	FILE_SORT_BY_SIZE         = "Size"
+	FILE_SORT_BY_USERNAME     = "Username"
+	FILE_SORT_BY_CHANNEL_NAME = "Name"
+
+	FILE_SORT_ORDER_ASCENDING  = "ASC"
+	FILE_SORT_ORDER_DESCENDING = "DESC"
+)
+
+// GetFilesOptions contains options for getting files
+type GetFilesOptions struct {
+	// UserIds optionally limits the files to those created by the given users.
+	UserIds []string
+	// ChannelIds optionally limits the files to those created in the given channels.
+	ChannelIds []string
+	// Since optionally limits files to those created after a specified time as Unix time in milliseconds.
+	Since int64
+	// IncludeDeleted includes deleted files if set.
+	IncludeDeleted bool
+	// Page optionally limits to the requested page of results.
```

Comment:
Let's promote `Page` and `PerPage` to be explicit parameters passed to the method, i.e. `GetFileInfos(page, perPage, options)`. We hadn't arrived at this conclusion when the first ticket was filed, but we don't want to support `GetAll`-type of methods, but force paging to limit the volume of reads that can occur. If we leave this in the `options` struct and instead set some defaults for paging, a developer might think that page was the full set of results without ever writing the necessary paging code.